spec/features/super_admin/view_organisation_page_spec.rb

```diff
@@ -2,11 +2,16 @@
   let(:organisation) { create(:organisation) }
 
   context 'when logged in as a super-admin' do
-    let(:location) { create(:location, organisation: organisation) }
+    let(:location_1) { create(:location, organisation: organisation, address: 'Aarry Street', postcode: 'HA7 3BL') }
+    let(:location_3) { create(:location, organisation: organisation, address: 'Carry Street', postcode: 'HA7 2BL') }
```

Comment:
parameters out of sequence, the big lint change incoming will complain about this.

Reply:
Also, they're not actually created here, so the order of creation is dictated within the `before` block where they're referenced

Reply:
noting also that the postcodes are different.
If they have no effect on ordering, please keep them the same so we reduce potential scope of this test